core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -257,6 +260,11 @@ object AdminUtils extends Logging {
           throw new InvalidTopicException("Topic \"%s\" collides with existing topics: %s".format(topic, collidingTopics.mkString(", ")))
         }
       }
+
+      //by default we make user that issues topic creation as the owner.
+      if (!config.containsKey(LogConfig.OwnersProp)) {
+        config.put(LogConfig.OwnersProp,  System.getProperty("user.name"))
+      }
```

Comment:
Not sure if we should get this from the user.name system property. Would it be better to just use the default owner if it's not specified explicitly?